docs/linux/zbd-api.md

```diff
@@ -302,17 +303,14 @@ enum blk_zone_cond {
 Under a device's normal operation, some of these conditions cannot result
 directly from host-initiated operations. These conditions are
 `BLK_ZONE_COND_OFFLINE` and `BLK_ZONE_COND_READONLY`. They can be set only by
-the device itself, to indicate zones with capabilities that have been limited by
-a hardware defect.
+the device itself, and are set to indicate zones with capabilities that have
+been limited by a hardware defect.
 
-Transitions to other conditions result from user operations, either write
-operations or zone management commands. Zone management commands can be issued
-by an application using the kernel *ioctl()* interface
-(see [*ioctl()* Commands](#ioctl-commands).
+User operations (either write operations or zone management commands) can result in transitions to other conditions. Applications that use the kernel *ioctl()* interface can issue zone managment commands. See [*ioctl()* Commands](#ioctl-commands) for more information about this.
```

Comment:
looks like a very long line...

Reply:
It sure does. I thought it might squeak through. Since it didn't, it's fixed.